apps/web/app/components/Gravatar.tsx

```diff
@@ -3,19 +3,21 @@ import CryptoJS from "crypto-js";
 import { AvatarImage } from "./ui/avatar";
 
 type GravatarProps = {
-  email: string;
+  email: string | undefined;
```

Comment:
email?: string

Reply:
I prefer using `email: string | undefined` instead of `email?: string` because the email field should always be included in the object, even if its value is undefined. 
With `email?: string`, the property can be completely omitted, which doesn't meet the requirement of ensuring the field is always present. 
Using `email: string | undefined` makes the intent clear that the property is required but can have an `undefined` value.